### ISSUE TITLE
Podcasts, some episodes report very long duration, even before any episodes are downloaded.

### ISSUE THREAD

**Issue by xekon** [ampache/ampache#2777]
## Description

When you first add a podcast normally all Time would show 0:00, however one of my podcasts does not do that for some of the episodes.

### Describe the bug

For the podcast "Darknet Diaries" many of the episodes are incorrectly reporting a VERY long duration.

### To reproduce

1. Go to 'Podcasts'
2. Click on 'Subscribe to Podcast'
3. Subscribe to https://feeds.megaphone.fm/darknetdiaries
4. See the huge timestamps on episodes: 2, 4, 5, 7, 8, 9, 10, 12, 13, 14, 15, 19, 23, 26, 30, 37, 44, 55, 72, 75

### Expected behavior

The episodes have not even downloaded yet, so I would assume they should all show a Time of 0:00

### Screenshots

![2021-02-23_23-43](https://user-images.githubusercontent.com/106644/108964929-fcfed900-7630-11eb-893e-f1407cf4b970.png)

## Environment

* Ampache version: Ampache 4.4.0-develop
* Web server + version: Nginx latest
* Server operating system: Ubuntu 20.04
* Client operating system: Ubuntu 20.04

### Client type

* Web

## Settings

`diff -u /var/www/ampache/config/ampache.cfg.php.dist /var/www/ampache/config/ampache.cfg.php | grep -E "^\+"`

```
+++ /var/www/ampache/config/ampache.cfg.php     2021-02-22 15:24:02.319463992 +0000
+http_host = "ampache.REDACTED.com"
+http_port = "443"
+web_path = ""
+local_web_path = "http://localhost:80"
+database_hostname = "/run/mysqld/mysqld.sock"
+database_port = ""
+database_name = "ampache"
+database_username = "www-data"
+database_password = ""
+database_charset = "utf8mb4"
+database_collation = "utf8mb4_unicode_ci"
+secret_key = "REDACTED"
+session_length = "3600"
+stream_length = "7200"
+remember_length = "604800"
+session_name = "ampache"
+session_cookielife = "0"
+session_cookiesecure = "0"
+getid3_tag_order = "id3v2,id3v1,vorbiscomment,quicktime,matroska,ape,asf,avi,mpeg,riff"
+additional_genre_delimiters = "[/]{2}|[/|\\|\|,|;]"
+licensing = "false"
+simple_user_mode = "true"
+artist_art_preferred_filename = "artist.jpg"
+album_art_min_width = "190"
+album_art_min_height = "190"
+art_order = "db,folder"
+wanted = "false"
+channel = "false"
+live_stream = "false"
+podcast = "true"
+debug = "true"
+debug_level = "5"
+log_filename = "%name.log"
+site_charset = "UTF-8"
+allow_public_registration = "false"
+cookie_disclaimer = "false"
+transcode_mp3 = "allowed"
```

## Logs

There does not appear to be any errors in the logs, if you cannot reproduce the problem I can provide the logs anyway if you request them.

**Comment by lachlan-00** [comment]
The time is pulled from the RSS and converted so there must be something about this one that's not parsed properly. Will add and check

**Comment by lachlan-00** [comment]
fixed in 7c80a1ded

Podcasts that report second length instead of a HH:MM:SS timestamp were converted into weird times when it should just be inserted as is.

**Comment by xekon** [comment]
Awesome! Thank You! Just tested, looks perfect!
